modules/csv/csv_test.go

```diff
@@ -322,7 +322,7 @@ func TestGuessDelimiter(t *testing.T) {
 		},
 		// case 3 - tab delimited
 		{
-			csv: "1	2",
+			csv:               "1\t2",
```

Comment:
This should be the only non-docs change in this PR.

Reply:
I hesitate to change this. Maybe it's a bug of old version or new version.

Reply:
This test passes on both 1.18 and 1.19

Reply:
Which one do you mean? The spaces between `csv:` and `"1 2"`, or the `\t` between `1` and `2`?

I think both of them are fine

Reply:
I think making this a `\t` instead of a literal tab is the right thing to do.